lib/modules/editor/screens/node.dart

```diff
@@ -69,11 +69,11 @@ class _NodeEditorScreenState extends State<NodeEditorScreen>
         ),
         const Divider(),
         ListTile(
+          leading: const Icon(Icons.notes_rounded),
```

Comment:
Here I think moving the icon outside of the field remove focus on the field by clicking the icon.

Reply:
hmm, I just felt that, semantically, the icon belongs to the list tile more than it does to the text field itself. Or even with semantics aside, it is better for consistency since not all list tiles contain text fields, and the styling was a little bit off and inconsistent.
To better clarify, look at the following screenshot from the currently deployed version:
![image](https://user-images.githubusercontent.com/11016151/162639102-11af16bc-4a7e-4e51-b417-bcd21db6644e.png)
The icons are not aligned horizontally, and the text field line extends to the icon, looking like a divider

Reply:
I agree. I've moved all the other suffix icons.